src/app/pages/task-calendar/cron/cron-form/cron-form.component.ts

```diff
@@ -50,7 +50,7 @@ export class CronFormComponent {
            type: 'input',
            name: 'cron_description',
            placeholder: T('Description'),
-           tooltip: T('Optional. Describe the new task.'),
+           tooltip: T('Describe the new cron job to add.'),
```

Comment:
"Enter a description of the new cron job."

Reply:
done